platform/admin/core/admin-core-impl/src/main/java/org/codice/ddf/admin/core/impl/SystemPropertiesAdmin.java

```diff
@@ -87,6 +87,17 @@
   private static final String LOCALHOST_DATA_MANAGER = "localhost-data-manager";
   private static final String DATA_MANAGER = "data-manager";
 
+  private String etcDir = System.getProperty(KARAF_ETC);
+  private String systemPropertyFilename = etcDir + File.separator + SYSTEM_PROPERTIES_FILE;
+  private String userPropertiesFilename = etcDir + File.separator + USERS_PROPERTIES_FILE;
+  private String userAttributesFilename = etcDir + File.separator + USERS_ATTRIBUTES_FILE;
+
+  private File systemPropertiesFile = new File(systemPropertyFilename);
+  private File userPropertiesFile = new File(userPropertiesFilename);
+  private File userAttributesFile = new File(userAttributesFilename);
+
+  private Properties systemDotProperties;
```

Comment:
✏️ I think these fields that you created can be simplified and used directly in some of the methods instead of via method arguments:
```java
private final String etcDir = System.getProperty(KARAF_ETC);

private final Properties customSystemDotPropertiesProperties = new Properties(new File(etcDir + File.separator + SYSTEM_PROPERTIES_FILE));
private final Properties usersDotPropertiesProperties = new Properties(new File(etcDir + File.separator + USERS_PROPERTIES_FILE));
private final File usersDotAttributesFile = new File(etcDir + File.separator + USERS_ATTRIBUTES_FILE);
```